stock_picking_analytic/models/stock_picking.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+    _name = "stock.picking"
+    _inherit = ["stock.picking", "analytic.mixin"]
+
+    original_analytic_distribution = fields.Json(
+        help="Technical field. Use to store the value of"
+        "analytic_distribution if there is no lines",
+    )
+    analytic_distribution = fields.Json(inverse="_inverse_analytic_distribution")
+    # Assign a dummy search arg to the field to avoid triggering the warning.
+    # Without this, use of move_ids_without_package in @api.depends() of the compute
+    # method would trigger a warning from
+    # https://github.com/odoo/odoo/blob/0f84366/odoo/fields.py#L808-L812
+    # pylint: disable=method-search
+    move_ids_without_package = fields.One2many(search="[]")
```

Comment:
The field is no longer a computed field but instead a One2many field with a domain. Maybe this party trick that served us well in the past can and should be removed in this version? See https://github.com/odoo/odoo/blob/3b42467/addons/stock/models/stock_picking.py#L450-L451

Reply:
Yees, is not longer to be user. Thanks for the review